closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -1706,6 +1725,32 @@ private static void checkProviderFormat(String providerInfo, boolean update) {
                 printStackTraceAndExit("Provider format is incorrect: " + providerInfo);
             }
         }
+
+        private void loadProfileCheck(String profileID) {
+            Enumeration<?> pNames = securityProps.propertyNames();
```

Comment:
Again, suggest a local to avoid repeated concatenation:
```java
            String profileDot = profileID + '.';
```

Reply:
When change to below:

                if (name.startsWith(profileID + '.')) {
                    if (name.contains(".jce.provider.")) {
                        profileCheckProviderNames.add(name);
                    } else {
                        profileCheckPropertyNames.add(name);
                    }
                }

There is no need to use a local variable for `profileID + '.'`.

Reply:
The benefit of the local would be that it will be evaluated once (total), instead of once per iteration of the loop.